fiorella/src/styles/layout/_body-message.scss

```diff
@@ -0,0 +1,76 @@
+.body-message {
+  background: $peppermind-color;
+  padding: 60px 20px;
+  text-align: center;
+
+  .title-message {
+    overflow: hidden;
+    font-family: $playfair-display;
+    font-weight: $font-light;
+
+    &:after, &:before {
+      background-color: $cerise-red;
+      content: "";
+      display: inline-block;
+      height: 1px;
+      position: relative;
+      vertical-align: middle;
+      width: 50%;
+    }
+
+    &:after {
+      left: 10px;
+      margin-right: -50%;
+    }
+
+    &:before {
+      margin-left: -50%;
+      right: 10px;
+    }
+  }
+
+  .image-message {
+    margin-top: 30px;
+    width: 150px;
+
+    @media #{ $tablet }, #{ $desktop } {
+      margin-top: 40px;
+      width: 200px;
+    }
+  }
+
+  .text-message {
+    margin: 10px 15px;
+    font-family: $merriweathersans;
+    font-weight: $font-light;
+    font-size: 16px;
+    line-height: 30px;
+
+    @media #{ $tablet } {
+      font-size: 14px;
+    }
+
+    @media #{ $desktop } {
+      font-size: 20px;
+      margin: 15px 20px;
+    }
+  }
+
+  .btn-read-more {
+    font-family: $playfair-display;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered display, font-family, font-weight, margin-top, padding, text-decoration